operators/dynatrace-operator/0.5.0/manifests/dynatrace-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1592 @@
+          - leases
+          verbs:
+          - get
+          - update
+          - create
+        - apiGroups:
+          - apps
+          resources:
+          - daemonsets
+          verbs:
+          - list
+          - watch
+        serviceAccountName: dynatrace-webhook
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - monitoring
+  - dynatrace
+  - oneagent
+  - activegate
+  - observability
+  links:
+  - name: Dynatrace Operator
+    url: https://github.com/Dynatrace/dynatrace-operator
+  - name: OpenShift Monitoring Info
+    url: https://www.dynatrace.com/technologies/openshift-monitoring
+  maintainers:
+  - email: support@dynatrace.com
+    name: Dynatrace LLC
+  maturity: alpha
+  provider:
+    name: Dynatrace LLC
+    url: www.dynatrace.com
+  version: 0.5.0
+  webhookdefinitions:
+  - admissionReviewVersions:
+    - v1beta1
+    containerPort: 443
+    conversionCRDs:
+    - dynakubes.dynatrace.com
+    deploymentName: dynatrace-webhook
+    generateName: cdynakubes.kb.io
+    sideEffects: None
+    targetPort: server-port
+    type: ConversionWebhook
+    webhookPath: /convert
+  - admissionReviewVersions:
+    - v1
+    - v1beta1
+    - v1alpha1
+    containerPort: 443
+    deploymentName: dynatrace-webhook
+    generateName: webhook.dynatrace.com
+    rules:
+    - apiGroups:
+      - dynatrace.com
+      apiVersions:
+      - v1beta1
+      operations:
+      - CREATE
+      - UPDATE
+      resources:
+      - dynakubes
+    sideEffects: None
+    targetPort: server-port
+    timeoutSeconds: 2
+    type: ValidatingAdmissionWebhook
+    webhookPath: /validate
+  - admissionReviewVersions:
+    - v1beta1
+    - v1
+    containerPort: 443
+    deploymentName: dynatrace-webhook
+    failurePolicy: Ignore
+    generateName: webhook.ns.dynatrace.com
+    reinvocationPolicy: IfNeeded
+    rules:
+    - apiGroups:
+      - ''
+      apiVersions:
+      - v1
+      operations:
+      - CREATE
+      - UPDATE
+      resources:
+      - namespaces
+      scope: Cluster
+    sideEffects: None
+    targetPort: server-port
+    timeoutSeconds: 2
+    type: MutatingAdmissionWebhook
+    webhookPath: /label-ns
+  - admissionReviewVersions:
+    - v1beta1
+    - v1
+    containerPort: 443
+    deploymentName: dynatrace-webhook
+    failurePolicy: Ignore
+    generateName: webhook.pod.dynatrace.com
+    reinvocationPolicy: IfNeeded
+    rules:
+    - apiGroups:
+      - ''
+      apiVersions:
+      - v1
+      operations:
+      - CREATE
+      resources:
+      - pods
+      scope: Namespaced
+    sideEffects: None
+    targetPort: server-port
+    timeoutSeconds: 2
+    type: MutatingAdmissionWebhook
+    webhookPath: /inject
+  skips: 
+    - dynatrace-operator.v0.4.2
+    - dynatrace-operator.v0.4.1
+    - dynatrace-operator.v0.3.0
```

Comment:
Please append `replaces: dynatrace-operator.v0.2.2`

Reply:
Wouldn't this break users clusters? As we changed the `installMode` from `OwnNamespace` to `AllNamespaces`. At least Redhat gave us the information, that Users anyway can not update from 0.2.2 to 0.5.0, because of that change and therefore they anyway have to deinstall and reinstall the operator again

Reply:
@mvalarh Hi, can you have a look on the comment above and share your thoughts ? :) Thanks!